skxray/core/correlation.py

```diff
@@ -198,8 +197,8 @@ def _init_state(num_levels, num_bufs, labels):
     )
 
 
-def lazy_multi_tau(image_iterable, num_levels, num_bufs, labels,
-                   internal_state=None):
+def lazy_one_time(image_iterable, num_levels, num_bufs, labels,
```

Comment:
Ah ok, that makes sense.  Can you basically add what you said about not wanting multi tau to the docstring of this function? That seems like very useful information.  Maybe add it at the top.  Something like:

```
"""Generator implementation of 1-time multi-tau correlation

If you do not want multi-tau correlation, set num_levels to 1 and
num_bufs to the number of images you wish to correlate
"""
```